src/mongoose_hooks.erl

```diff
@@ -899,6 +902,14 @@ can_access_identity(HostType, Room, User) ->
 can_access_room(HostType, Room, User) ->
     run_hook_for_host_type(can_access_room, HostType, false, [HostType, Room, User]).
 
+-spec get_room_affiliations(Acc, Room) -> Result when
```

Comment:
Why is this hook not used anywhere?

Reply:
Because I'll use it on an upcoming PR. But yeah, maybe I should have added the hook on that other PR 🤔

Reply:
No problem, I just wanted to make sure nothing was missing from the PR.